### ISSUE TITLE
Suddenly getting: Unable to download dependency

### ISSUE THREAD

**Issue by rdifrango** [openrewrite/rewrite-maven-plugin#400]
On a recent upgrade, I've suddenly started getting the following error message when trying to run OpenRewrite, despite the fact that normal builds which work just fine.

```
[WARNING] Unable to download dependency info.cukes:cucumber-html:0.2.3 from the following repositories :
  - file:///Users/rdifrango/.m2/repository/
  - https://repo.maven.apache.org/maven2
```

**Comment by rdifrango** [comment]
And I'll add that in my particular case we do have an internal mirror which Maven is exclusively using where as OpenRewrite seems to be going direct to maven central in some cases.

**Comment by tkvangorder** [comment]
@rdifrango What version of the plugin were you using and have you tried the latest release (4.31.3)?

**Comment by rdifrango** [comment]
actually, I can close this as it seems as if `4.31.1` solved this, but I still am running into #399 with it.

**Comment by tkvangorder** [comment]
OK, I will close this one and try to get some time to look at 399. Thanks for your patience!
